Configure

```diff
@@ -76,8 +76,12 @@ my $usage="Usage: Configure [no-<cipher> ...] [enable-<cipher> ...] [-Dxxx] [-lx
 
 # Minimum warning options... any contributions to OpenSSL should at least get
 # past these.
-
-my $gcc_devteam_warn = "-DPEDANTIC -DREF_DEBUG -DDEBUG_UNUSED -DBIO_DEBUG"
+#
+# DEBUG_UNUSED enables __owur (warn unused result) checks.
+# PEDANTIC conditionally enables different codepaths in low-level crypto code
+# to get past PEDANTIC. Contributions to OpenSSL should therefore also pass
+# without PEDANTIC.
```

Comment:
PEDANTIC also needed for some compilers that can't handle files that are "empty"  Not sure if that's worth calling out.